packages/cli-repl/test/repl-helpers.ts

```diff
@@ -71,10 +71,7 @@ async function waitEval(bus: MongoshBus) {
 }
 
 async function waitCompletion(bus: MongoshBus) {
-  await Promise.race([
```

Comment:
This timeout was just always going to be flaky and pressing tab twice doesn't necessarily mean it completed a potentially slow operation anyway. tabtab() always relied on 2 * 250ms being enough.